apps/prairielearn/src/lib/score-upload.ts

```diff
@@ -1,34 +1,46 @@
-// @ts-check
-const _ = require('lodash');
-const streamifier = require('streamifier');
-const csvtojson = require('csvtojson');
-const sqldb = require('@prairielearn/postgres');
+import * as _ from 'lodash';
+import * as streamifier from 'streamifier';
+import csvtojson = require('csvtojson');
+import * as sqldb from '@prairielearn/postgres';
+import { z } from 'zod';
 
-const { createServerJob } = require('./server-jobs');
-const manualGrading = require('./manualGrading');
+import { createServerJob } from './server-jobs';
+import * as manualGrading from './manualGrading';
+import { IdSchema } from './db-types';
 
 const sql = sqldb.loadSqlEquiv(__filename);
 
+const assessment_info_schema = z.object({
```

Comment:
Schemas conventionally use PascalCase:

```suggestion
const AssessmentInfoSchema = z.object({
```